process/testing/winfv/setup-fv-capz.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+# This script sets up a k8s cluster with one Linux node and one Windows node and run windows FV test.
+# Usage - OS Felix fv, following variables need to be set
+#         AZURE_SUBSCRIPTION_ID
+#         AZURE_TENANT_ID
+#         AZURE_CLIENT_ID
+#         AZURE_CLIENT_SECRET
+#
+
+GIT_VERSION=$(git describe --tags --dirty --long --always --abbrev=12)
+CALICO_HOME=$(cd ../../../; pwd)
```

Comment:
How about this, makes it more robust in case this script is called from another dir:
```suggestion
CALICO_HOME=$(cd $(dirname $0)/../../../; pwd)
```